Makefile

```diff
@@ -148,6 +149,9 @@ help: ## Display this help.
 	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n"} /^[a-zA-Z_0-9-]+:.*?##/ { printf "  \033[36m%-15s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
 
 ##@ Development
+# golang version check
+golang_check: golang_check ## Ensure the golang version meets the requirements
```

Comment:
Recursion?
```suggestion
golang_check: ## Ensure the golang version meets the requirements
```

Reply:
sorry.. but you are incorrect sir..   
the first one is for the make command, the second is for the help parser